### PR TITLE
[MS] Added lostFocus event

### PR DESCRIPTION
**Pull request by fabienSvtr**

### CODE REVIEW

File: lib/components/ms-input/MsInput.vue [hunk 1]
```diff
@@ -60,12 +61,14 @@ const props = defineProps<{
...
 const errorMessage: Ref<Translatable> = ref('');
 const validity = ref(Validity.Intermediate);
-const lostFocus = ref(false);
+const lostFocus = ref(true);
 
 const emits = defineEmits<{
   (e: 'update:modelValue', value: string): void;
   (e: 'change', value: string): void;
   (e: 'onEnterKeyup', value: string): void;
+  (e: 'onFocusLost', value: boolean): void;
```
Comment (Max-7) [inline lib/components/ms-input/MsInput.vue:70]: You don't need booleans here.
If you have a signal like `focusChanged`, you can add a boolean `isFocused`.
Here, for `onFocusLost`, the boolean will always be true, and for `onFocus`, the boolean will always be false, which makes them useless.

File: lib/components/ms-input/MsAddressInput.vue [hunk 2]
```diff
@@ -54,8 +55,20 @@ defineExpose({
 const geoapifyApi = new GeoapifyAPI(props.geoapifyApiKey);
 const addressesFound = ref<Array<Address>>([]);
 const address = ref('');
+const lostFocus = ref(true);
```
Comment (Max-7) [inline lib/components/ms-input/MsAddressInput.vue:58]: `lostFocus` is weird for a variable name. You initialize it to true, but it's incorrect: focus has not been lost yet.
Why not rename it to `isFocused` instead? Init it to false, set it to false in `handleFocusLost`, to true in `handleFocus`, and the condition become `v-if="addressesFound.length > 0 && isFocused`.

File: lib/components/ms-input/MsAddressInput.vue [hunk 3]
```diff
@@ -54,8 +55,20 @@ defineExpose({
...
 let querying = false;
 
+function handleFocusLost(): void {
+  // Wait a bit to let the click event on the dropdown to be processed
+  setTimeout(() => {
```
Comment (Max-7) [inline lib/components/ms-input/MsAddressInput.vue:63]: What's the point of this timeout? Doesn't seem to change anything if I remove it. And code that relies on a 100ms timeout is very probably code that is bugged.

**Review by Max-7 — CHANGES_REQUESTED** [review]

File: lib/components/ms-input/MsAddressInput.vue (hunk 3 above)
Comment (fabienSvtr) [reply lib/components/ms-input/MsAddressInput.vue:62]: If we don't define a timeout, we cannot select an address, so 100ms is fine to make it works.

**Review by Max-7 — APPROVED** [review]

**Pull request merged**